changelog/60880.fixed

```diff
@@ -0,0 +1 @@
+Added info to docs for iptables `- name_:` vs `- name:`.
```

Comment:
changelog files are not required for doc changes.

Reply:
:fire: gone now :+1: